webapp/app/forms/steps/lotse/fahrkostenpauschale.py

```diff
@@ -0,0 +1,152 @@
+        return 4500
+    elif disability_degree is not None and disability_degree >= 80 or (has_merkzeichen_g and disability_degree >= 70):
+        return 900
+
+    return 0
+
+
+class StepFahrkostenpauschale(LotseFormSteuerlotseStep):
+
+    def get_overview_value_representation(self, value):
+        result = None
+
+        if value == 'yes':
+            result = str(self.get_fahrkostenpauschale()) + ' ' + _('currency.euro')
+        elif value == 'no':
+            result = _('form.lotse.summary.not-requested')
+
+        return result
+
+    def get_fahrkostenpauschale(self):
+        """
+            Get the amount of Fahrkostenpauschale for this step. Should be implemented for each step.
+        """
+        pass
+
+
+class StepFahrkostenpauschalePersonA(StepFahrkostenpauschale):
+    name = 'person_a_requests_fahrkostenpauschale'
+    section_link = SectionLink('mandatory_data', StepFamilienstand.name, _l(
+        'form.lotse.mandatory_data.label'))
+
+    preconditions = [HasDisabilityPersonAPrecondition, HasMerkzeichenPersonAPrecondition]
+
+    class InputForm(SteuerlotseBaseForm):
+        person_a_requests_fahrkostenpauschale = SelectField(
+            choices=[('yes', 'yes'), ('no', 'no')],
+            render_kw={'data_label':  _l(
+                'form.lotse.request_fahrkostenpauschale.data_label')},
+            validators=[InputRequired(_l('validate.input-required'))])
+
+    @classmethod
+    def get_label(cls, data=None):
+        return ngettext('form.lotse.person_a.request_fahrkostenpauschale.label', 'form.lotse.person_a.request_fahrkostenpauschale.label',
+                        num=get_number_of_users(data))
+
+    def render(self):
+        props_dict = FahrkostenpauschaleProps(
+            step_header={
+                'title': ngettext('form.lotse.person_a.request_fahrkostenpauschale.title', 'form.lotse.person_a.request_fahrkostenpauschale.title',
+                                  num=get_number_of_users(self.stored_data))
+            },
+            form={
+                'action': self.render_info.submit_url,
+                'csrf_token': generate_csrf(),
+                'show_overview_button': bool(self.render_info.overview_url),
+            },
+            fahrkostenpauschale_amount=self.get_fahrkostenpauschale(),
+            fields=form_fields_dict(self.render_info.form),
+            prev_url=self.render_info.prev_url
+        ).camelized_dict()
+
+        return render_template('react_component.html',
+                               component='FahrkostenpauschalePersonAPage',
+                               props=props_dict,
+                               form=self.render_info.form,
+                               header_title=_('form.lotse.header-title'))
+
+    def get_fahrkostenpauschale(self):
+        return calculate_fahrkostenpauschbetrag(
+            has_pflegegrad=self.stored_data.get('person_a_has_pflegegrad', False),
```

Comment:
This can't work... person_a_has_pflegegrad is a 'yes, 'no' string

Reply:
Valid point. This is changed in #538 . We should maybe just take that implementation here.